### ISSUE TITLE
Missing support for Symfony 7

### ISSUE THREAD

**Issue by tarlepp** [mark-gerarts/automapper-plus-bundle#26]
It would be great if this amazing library supports also new Symfony 7.

**Comment by mark-gerarts** [comment]
Hi @tarlepp, I added support for Symfony 7 on a separate branch, `feature/symfony-7`. Once that's tested for a bit, I'll bump the version.

**Comment by th-lange** [comment]
I still get deprecation infos when using the bundle (on Symfony 6.4)

This is not fixed in the bundle if I see correctly.

So maybe we can add a "void" here?


```
class AutoMapperPlusExtension extends Extension
{
    private const DEFAULT_OPTIONS_CONFIGURATOR_SERVICE_ID = 'automapper_plus.default_options_configurator';


    /**
     * @inheritdoc
     */
    public function load(array $configs, ContainerBuilder $container)
    {

```

Deprecation warning:

```
[2023-12-12T16:14:13.083778+00:00] php.INFO: User Deprecated: Method "Symfony\Component\DependencyInjection\Extension\ExtensionInterface::load()" might add "void" as a native return type declaration in the future. Do the same in implementation "AutoMapperPlus\AutoMapperPlusBundle\DependencyInjection\AutoMapperPlusExtension" now to avoid errors or add an explicit @return annotation to suppress this message. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Method \"Symfony\\Component\\DependencyInjection\\Extension\\ExtensionInterface::load()\" might add \"void\" as a native return type declaration in the future. Do the same in implementation \"AutoMapperPlus\\AutoMapperPlusBundle\\DependencyInjection\\AutoMapperPlusExtension\" now to avoid errors or add an explicit @return annotation to suppress this message. at /srv/app/vendor/symfony/error-handler/DebugClassLoader.php:337)
[stacktrace]
```

**Comment by th-lange** [comment]
Same goes for:

```
class ConfigurationLoaderPass implements CompilerPassInterface {

     // ....

    /**
     * @inheritdoc
     */
    public function process(ContainerBuilder $container)

```

**Comment by shoaibgitrepo** [comment]
Hi @mark-gerarts, is this ready on Composer for Symfony 7?

**Comment by mark-sentigrate** [comment]
Thanks to #27, this should be fixed on master. Can you confirm this is working? I'll tag it as a new release then.

**Comment by mark-gerarts** [comment]
Thanks to #27, this should be fixed on master. Can you confirm this is working? I'll tag a new version then.

**Comment by mark-gerarts** [comment]
Changes are available starting from release `1.5.0`.
